P5/Exemplars/tei_jtei.odd

```diff
@@ -237,15 +239,20 @@
             <item>The title of your article (in <code>fileDesc/titleStmt/title[@type='main']</code>).</item>
             <item>Author information for each author (in <code>fileDesc/titleStmt/author</code>).
               Provide one <gi>author</gi> element for each author. Encode the names as shown, using
-                <gi>forename</gi> and <gi>surname</gi>, inside <gi>name</gi>. Then provide a brief
+                <gi>forename</gi> and <gi>surname</gi>, inside <gi>name</gi> if possible. 
+<!--       [PL]       forename and surname are very much western habits. Ethiopian colleagues do not have this habit, for example.
+              why not allow all what is possible in name? I also do not see constraints here in the ODD about this.
+              -->
```

Comment:
@PietroLiuzzo I'm open to the idea, but the (equally Western-centric) Lodel publication platform does expect surnames and forenames for all contributors (https://tei-openedition.readthedocs.io/en/latest/tei.teiHeader.html#auteurs-editeurs-traducteurs)

Reply:
the example there has name without further specification, which would be already good 
````xml
<author>
<name>Marin Dacos</name>
</author>
````
this would be ok for us, and I did not find a rule in the schema requiring that.

Reply:
I noticed, too... and have tested with the Lodel publication platform, which does seem to accept just `<name>`, and will then index the last token as surname, and all the rest as forename. So I can confirm `<forename>` and `<surname>` are not mandatory for the publication system, although they will provide for better control over the indexing of names in the Lodel publication platform (with all apologies for the non-descriptive markup angle of these remarks).

On further investigation, the jTEI schema doesn't require `<forename>` and `<surname>` either, so it's just the prose description that is over-specified. I do think it's worth mentioning that _if name components can be distinguished,_ they should preferably be a) placed in the order "forename - surname", and b) tagged explicitly with `<forename>` and `<surname>` .

Reply:
I'd be in favour of making `<forename>` and `<surname>` mandatory. I'm constantly finding holes in transformation code that assumes the surname is a single token when quite frequently it's a double-barrelled name with no hyphen (Ralph Vaughan Williams, Andrew Lloyd Webber, John Maynard Smith, Kristin Scott Thomas, Helena Bonham Carter).

Reply:
I like the way of putting it of @rvdb last comment !

Reply:
I would not mandate or assume order of surname forename, unless you think `Zedong Mao` is how `Mao Zedong` should appear in jTei. There is a Stylesheets issue discussing this already.

Reply:
The fork will have to be refreshed too, because the jTEI ODD seems to have moved on since the PR was originally created.